pyscan/main.py

```diff
@@ -42,6 +46,12 @@ def print_debug(msg):
     if DEBUG:
         print(msg)
 
+def send_sensor_data(name, timeout):
+    while(pybytes):
+        pybytes.send_virtual_pin_value(True, 2, lt.light())
+        pybytes.send_virtual_pin_value(True, 3, li.acceleration())
```

Comment:
Hi @Aditya18Jain 👋 

Could you change these to simple `print()`? That's what we use in the other examples as well. By default, these examples should run on every devices/firmware type. I'll merge this after that's changed. Thanks!

Reply:
replaced the code with suggested changes.